api/FederatedCredential.json

```diff
@@ -133,6 +133,51 @@
             "deprecated": false
           }
         }
+      },
+      "protocol": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FederatedCredential/protocol",
+          "support": {
+            "chrome": {
+              "version_added": null
```

Comment:
The change that enabled this is no longer available to me in GitHit history. (There was a file name change two years ago and history for the old file is no longer available in https://cs.chromium.org.) The [Chromium blog](https://blog.chromium.org/2016/04/chrome-51-beta-credential-management.html) and the [Chrome Status entry](https://www.chromestatus.com/features/5026422640869376) agree on Chrome 51. The [last commit of this interface before 51](https://chromium.googlesource.com/chromium/src/+/64ed1db25af23c6e04c9c24fc22f2688ce306f77) show that protocol was part of the API.